packages/salesforcedx-vscode-soql/src/telemetry/index.ts

```diff
@@ -23,10 +23,10 @@ export const stopTelemetry = (): Promise<void> => {
   return Promise.resolve();
 };
 
-export interface TelemetryModelJson extends JsonMap {
+export type TelemetryModelJson = JsonMap & {
```

Comment:
@daphne-sfdc I don't believe JsonMap is not necessary.

Reply:
@peternhale The double negative means you think it is necessary and what's currently there is fine?

Reply:
not necessary